.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Pull request
+about: Use this template to open pull requests
+title: '<title>'
+labels:
+assignees: 'KateFriedman-NOAA', 'WalterKolczynski-NOAA'
+
+---
+
+**Description**
+
+<!-- Please include relevant motivation and context. -->
+<!-- Please include a summary of the change and which issue is fixed. -->
+<!-- List any dependencies that are required for this change. -->
+
+<!-- Please provide reference to the issue this pull request is addressing. -->
+<!-- For e.g. Fixes #IssueNumber -->
+
+**Type of change**
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+**How Has This Been Tested?**
+
+<!-- Please describe the tests that you ran to verify your changes and on the platforms these tests were conducted. -->
+<!-- Provide instructions so we can reproduce. -->
+<!-- Please also list any relevant details for your test configuration -->
+
+<!-- Use the following as a guide to list your tests and delete options that are not relevant. Expand as necessary. -->
+<!--
+- [ ] Cycled test on Orion
+- [ ] Forecast only test on Hera
+-->
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
```

Comment:
Given we don't have formal tests set up yet, might want to remove these two for now.

Reply:
Sounds good.